controllers/topics-controllers

```diff
@@ -41,6 +42,10 @@ function getArticles(req, res, next) {
 
 function getCommentByArticleId(req, res, next) {
   const articleIndex = req.params.article_id;
+
+  // need to change the code to check for articles
+  // without comments (now is comming as error 404)
+
```

Comment:
please remove comments! :) We should only push stuff up to github that we would be happy for senior devs/ the public/ recruiters to see :)